plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -124,6 +124,17 @@ func init() {
 		},
 	})
 	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "expand-controller"},
+		Rules: []rbac.PolicyRule{
+			rbac.NewRule("update").Groups(legacyGroup).Resources("persistentvolumes").RuleOrDie(),
+			rbac.NewRule("update").Groups(legacyGroup).Resources("persistentvolumeclaims/status").RuleOrDie(),
```

Comment:
we try to keep roles defined so that if the controller was run solo in a pod, the role would be sufficient for its needs. I'm assuming this makes use of list/watch on PVs, and if `update` is granted, `get` and `patch` are typically allowed as well.

Reply:
I had to grant permissions for reading secrets, endpoints, services as well to expand-controller's client.  It should have enough permissions to run standalone now.

Reply:
what client was it using to get secrets before? the attach/detach controller doesn't have secret access...

Reply:
Attach/Detach controller doesn't handle glusterfs. The reason, we had to add secret here is because there was a change made for glusterfs Storageclass so that glusterfs credentials are no longer directly available in SC (you should remember this change!). 

So credentials for heketi are stored in another namespace and this client must be able to read them - to be able to resize glusterfs volumes.

Again to reiterate - since glusterfs is not a attachable volume type, attach/detach controller doesn't do anything with it.  Originally when I tested this PR, I didn't had rbac enabled cluster but other people who have contributed to this PR ( @wongma7 , @humblec) did. Now, in addition to them - I have tested with rbac enabled cluster as well and these permissions here are needed.